src/upstream/teleEmitter.ts

```diff
@@ -239,10 +239,11 @@ export class TeleReporterEmitter implements ReporterV2 {
 
   _serializeAttachments(attachments: reporterTypes.TestResult['attachments']): teleReceiver.JsonAttachment[] {
     return attachments.map(a => {
+      const omit = this._emitterOptions.omitBuffers && a.name !== '_error-context';
```

Comment:
I am confused about this one, to be honest.

Reply:
We seem to enable `omitBuffers` for a reason unbeknownst to me. I didn't want to disable it across the board, so I only disabled it for the attachment I care about.

Reply:
we resolved this in https://github.com/microsoft/playwright-vscode/pull/627.